### PR TITLE
fix Issue 19491 - ICE (segfault) when initializing scope variable wit…

### PR DESCRIPTION
**Pull request by WalterBright**
…h shared type

### CODE REVIEW

**Review by RazvanN7 — APPROVED** [review]

**Comment by RazvanN7** [comment]
This should target stable.

File: test/compilable/test19491.d [hunk 1]
```diff
@@ -0,0 +1,9 @@
+// https://issues.dlang.org/show_bug.cgi?id=19491
+
+class Foo {
```
Comment (jacob-carlborg) [inline test/compilable/test19491.d]: Curly braces on their own lines.

Comment (wilzbach) [reply test/compilable/test19491.d]: We don't really enforce/require the style guide for test files as they get seldomly looked at. So imho it's not worth bothering about this here.

Comment (jacob-carlborg) [reply test/compilable/test19491.d]: In my opinion one should take as good care of the tests as the rest of the code.

Comment (thewilsonator) [reply test/compilable/test19491.d]: Fixed.

**Comment by thewilsonator** [comment]
> This should target stable.

Done.

**Pull request merged**